example/spec/parallel/moveGranules/MoveGranulesTagsAndZeroByteFileSpec.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
```

Comment:
Not all of this test is new, I combined new tests with the test file below that is being deleted/renamed with some test setup changes.

Reply:
If you do `git mv filename newfilename`, it will preserve the history.
Maybe you can rename it back and checkin, then rename again.

Reply:
I attempted your suggestion. When I pushed and it was the old file name, it showed here as the one file with all the changes. I used the command to commit and push, and it showed here as 2 different files again even though before the commit it showed as a rename change.

Reply:
We can use the original filename.

Reply:
But the file is testing more than just Tags, it is also testing the ZeroByteFile. Using the old filename seems incomplete or misleading, doesn't it?

Reply:
I think it's better than losing the git history.  We can rename it to a more generic name later